iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergOutputCommitter.java

```diff
@@ -307,7 +307,7 @@ private void commitTable(FileIO io, ExecutorService executor, JobContext jobCont
     if (catalogName != null) {
       catalogProperties.put(InputFormatConfig.CATALOG_NAME, catalogName);
     }
-    Table table = Catalogs.loadTable(conf, catalogProperties);
+    Table table = Catalogs.loadTableSkipCache(conf, catalogProperties);
```

Comment:
Should it be `Catalogs.invalidate` if the catalog implements `CachingCatalog` interface?
This way we break the compatibility with other Catalogs.

Reply:
I'm not sure I understand your point here. Could you please add a bit more details?